mllib/src/main/scala/org/apache/spark/ml/feature/IDF.scala

```diff
@@ -151,6 +151,15 @@ class IDFModel private[ml] (
   @Since("2.0.0")
   def idf: Vector = idfModel.idf.asML
 
+  /** Returns the DF */
+  @Since("3.0.0")
+  def df: Array[Long] = idfModel.df
```

Comment:
I know this parallels 'idf' but 'df' often means 'dataframe'. I'd just call this documentFrequency or something

Reply:
Renamed it as `docFreq`. Thanks.